### DISCUSSION TITLE
filter your own boosts from your home timeline if you filter boosts from your home timeline

### DISCUSSION THREAD

**Discussion by aescling** [cat-KIBI-glitch/mastodon#20] [Ideas]
yes. if you filter boosts from your home timeline, your own boosts still appear in your home timeline.

Plural Café filters out your own boosts, and i much prefer my home timeline working this way

**Comment by marrus-sh** [comment]
The main reason why this isn’t default behaviour is because the home timeline (generally) provides feedback on user action. You make a post, you boost something, you do a reply, it all appears in the home timeline as confirmation that the action completed successfully.

So I don’t think we should eliminate this behaviour completely, but I could see setting it behind an additional toggle, like :—

> ☑️ Including your own boosts

**Reply by aescling** [reply]
that makes sense and i like your suggestion for the toggle
